release/release.gradle

```diff
@@ -433,6 +430,17 @@ class ChangeLogFile {
         releaseNotes.addEmptyLine()
         return releaseNotes.notes
     }
+
+    private static getVersionId(jsonReleaseNotes, String restReleaseVersion) {
+        def fixVersions = jsonReleaseNotes.issues.get( 0 ).fields.fixVersions
+
+        for ( def fixVersion : fixVersions ) {
+            if ( fixVersion.name.equals( restReleaseVersion ) ) {
+                return fixVersion.id
+            }
+        }
+        throw new GradleException( "Unable to determine the version id of the current release." )
+    }
```

Comment:
There is more idiomatic Groovy way to achieve the logic as
```
return Optional.ofNullable( fixVersions.find { it.name.equals( restReleaseVersion ) } ).orElseThrow{new GradleException( 'Unable to determine the version id of the current release.' ) }
```
though not a big deal and feel free to ignore my comment.